<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>password_forget</class>
 <widget class="QMainWindow" name="password_forget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget
{
	background:url(:/source/back_1.jpg);
}
*{
font: 9pt &quot;Bauhaus 93&quot;;
font-size:24px;
}
QToolButton{
background:pink;
border-radius:60px;
}
QPushButton
{
background:white;
border-radius:15px;
font: 12pt &quot;华文琥珀&quot;;
}
QPushButton:hover
{
background:pink;
border-radius:15px;
font: 12pt &quot;华文琥珀&quot;;
}
QLineEdit
{
background:rgba(255,255,255,0.5);
border:none;
color:white;
font: 12pt &quot;华文琥珀&quot;;
background:transparent;
border-bottom:1px solid #717072;
}
QFrame{
background:rgba(255,255,255,0.6);
border-radius:15px;
}
QLabel
{
background:transparent;
color:black;
font:20pt &quot;华文琥珀&quot;;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>忘记密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_email">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>输入你的邮箱号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>输入你的用户名</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>410</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>查看密保问题</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{	
	color:gray;
	font-size:18px;
}</string>
    </property>
    <property name="text">
     <string>两者仅需输入一个</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>160</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_answer">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>290</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>输入密保答案</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>560</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>确认修改</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_new_password">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>430</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>输入新密码</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
